websocket.el

```diff
@@ -4,7 +4,7 @@
 
 ;; Author: Andrew Hyatt <ahyatt@gmail.com>
 ;; Keywords: Communication, Websocket, Server
-;; Version: 1.5
+;; Version: 1.6
```

Comment:
I do that when I release (although I will likely do that after we straighten out all these Windows issues).  But I basically want to run tests on my system to make sure things still work as they used to.